Tests/NIOHTTP2Tests/SimpleClientServerTests.swift

```diff
@@ -1230,4 +1230,90 @@ class SimpleClientServerTests: XCTestCase {
         XCTAssertNoThrow(try self.clientChannel.finish())
         XCTAssertNoThrow(try self.serverChannel.finish())
     }
+
+    func testManyConcurrentInactiveStreams() throws {
+        let maxCachedClosedStreams = 1024
+
+        // Begin by getting the connection up.
+        try self.basicHTTP2Connection(maxCachedClosedStreams: 1024)
```

Comment:
use the `maxCachedClosedStreams` variable here?

Reply:
Fixed.